rewiringamerica/embed.rewiringamerica.org
# Add "select a project" card

## Description

Stop selecting a project by default upon getting results, and show a
card telling the user to select one.

As part of this, the "no incentives" card is no longer needed. You
can't select a project that has no incentives, and there is always at
least one project that has incentives (because some of the IRA tax
credits are available to everyone) so there's no need for that empty
state anymore. This unlocks a good bit of code cleanup.

### Follow-up

- <s>Wait for an SVG version of the image to go in this card; the PNG is a
  placeholder.</s>

- <s>Spanish translations</s>

## Test Plan

- Submit the form and see this card; make sure it looks good on all
  widths, and the "select project" placeholder shows up in the dropdown.

- Select a project, make sure incentives show properly. Make sure
  there's no blank item in the menu (i.e. you can't go back to this
  card after selecting a project).

- Submit the form again with different inputs; make sure the project
  you selected before is still selected, unless it now has no
  incentives, in which case you should be back on the card. (To make
  the latter case happen, submit a low income and choose "cooking",
  then submit a high income.)
